src/Symfony/Bundle/FrameworkBundle/Resources/config/session.xml

```diff
@@ -48,11 +48,17 @@
             <argument type="service" id="session.storage.metadata_bag" />
         </service>
 
-        <service id="session.handler.native_file" class="Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeFileSessionHandler">
-            <argument>%session.save_path%</argument>
+        <service id="session.handler.native_file" class="Symfony\Component\HttpFoundation\Session\Storage\Handler\SessionHandlerProxy">
+            <argument type="service">
+                <service class="Symfony\Component\HttpFoundation\Session\Storage\Handler\NativeFileSessionHandler">
```

Comment:
for some obscure reason, the native `SessionHandler` doesn't implement `SessionUpdateTimestampHandlerInterface`
we have to proxy it to add strict mode and lazy writes...